platforms/hyperledger-besu/configuration/roles/create/prometheus/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+# Fetching the prometheus URLs
+- name: Fetching the prometheus URLs
+  include_tasks: nested_loop.yaml
+  vars:
+    members: "{{service}}" 
+    org_name: "{{org_loop.name}}"
+
+# Install prometheus via helmchart
+# - name: Install  prometheus server
+#   shell: |
+#     KUBECONFIG={{ kubernetes }} helm upgrade --install {{ name }}-prom --namespace={{ component_ns }}  \
+#     {{ playbook_dir }}/../../../platforms/shared/charts/prometheus \
+#     -f  {{ playbook_dir }}/../../../platforms/hyperledger-besu/releases/dev/{{name}}-bes/{{name}}-prometheus.yaml
+
+# Install prometheus via helmchart
+- name: Install  prometheus server
+  shell: |
+    KUBECONFIG={{ kubeconfig_path }} helm repo add prometheus-community https://prometheus-community.github.io/helm-charts --force-update
```

Comment:
Shall we start using Helm module for chart installations? @suvajit-sarkar 
https://docs.ansible.com/ansible/latest/collections/kubernetes/core/helm_module.html (The first example here is for the prometheus itself)

Reply:
yes it makes sense to use modules